internal/kubernetes/nodefilters.go

```diff
@@ -41,6 +42,22 @@ func NewNodeLabelFilter(labels map[string]string) func(o interface{}) bool {
 // NewNodeConditionFilter returns a filter that returns true if the supplied
 // object is a node with any of the supplied node conditions.
 func NewNodeConditionFilter(ct []string) func(o interface{}) bool {
+	var specifiedConditions [][]string
+	//make parsing once
+	for _, t := range ct {
+		kv := strings.SplitN(t, "=", 2)
+		// default values as if we specify only condition type e.g. OutOfMemory
+		tp := t
+		st := "True"
```

Comment:
https://godoc.org/k8s.io/api/core/v1#ConditionStatus

There's a constant in the client-go package we could use for this.